Tests/Realm.Tests/Sync/SynchronizedInstanceTests.cs

```diff
@@ -444,21 +444,25 @@ public void DeleteRealmWorksIfCalledMultipleTimes()
         }
 
         [Test]
-        public void DeleteRealm_afterSessionDisposed([Values(true, false)] bool singleTransaction)
+        public void DeleteRealm_AfterDispose_Succeeds([Values(true, false)] bool singleTransaction)
```

Comment:
This test name was not following our naming guidelines and was a bit unclear. The new name is not much better, so I added some comments in the test body to clarify it.

Reply:
Is the `succeeds` part here redundant? I assume it refers to the deletion, not the test itself but would that mean we should always add `succeeds` and `fails` to all tests to indicate of they test a positive or a negative case?

Reply:
Generally, I've been trying to name tests like *method*_*when-something*_*result*. I guess here a more precise name would have been `DeletesFiles` rather than `Succeeds` - I guess I was a little lazy when coming up with the name. That being said, our tests are a bit of a mixed bag when it comes to naming conventions, so it doesn't matter that much.

Reply:
👍